VectorMap: update Dynamic Viewport description (#2245)
DevExpress/devextreme-demos: JSDemos/Demos/VectorMap/DynamicViewport/description.md

> This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. The map changes the visible area so that the chosen continent is displayed optimally as a result.

Reviewer: ```suggestion
This demo illustrates how to change the map's viewport. Use the drop-down menu under the map to choose a continent and change the visible area so that the chosen continent is displayed optimally.
```

> This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. In response to that, the map changes the visible area so that the chosen continent is displayed optimally. 

Reviewer 2: ```suggestion
This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. The map changes the visible area so that the chosen continent is displayed optimally as a result.
```

> To implement this functionality, call the [viewport(viewportCoordinates)](/Documentation/ApiReference/UI_Components/dxVectorMap/Methods/#viewportviewportCoordinates) method every time the drop-down box value changes. You can also implement the [onCenterChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onCenterChanged) and [onZoomFactorChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onZoomFactorChanged) functions. These functions place the current values of the map’s center and zoom factor in the text boxes under the map.

Reviewer 2: Why do we need to implement the mentioned functions? To display the current values in text boxes under the map. If you try to make your sentences very short, it may lead to awkward repetitions...

```suggestion
To implement this functionality, call the [viewport(viewportCoordinates)](/Documentation/ApiReference/UI_Components/dxVectorMap/Methods/#viewportviewportCoordinates) method every time the drop-down box value changes. You can also implement the [onCenterChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onCenterChanged) and [onZoomFactorChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onZoomFactorChanged) functions to display the map center's coordinates and the current zoom factor in text boxes under the map.
```

> To specify the [control bar](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/) visibility, enable or disable [panVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#panVisible) and [zoomVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#zoomVisible) properties. You can toggle switches under the map to see how it affects the control bar. 

Reviewer 2: To specify the visibility. Yes, in theory, you can say like that. However, we seek to be as close to real-life tasks as possible. To show or hide the control bar...

If you use property names along with the word "property" or "properties", you surely need a definite article.

> to see how **it** affects the control bar.

"It" what? While it's "obvious", we still should seek to be more specific at everything.

```suggestion
To show or hide the [control bar](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/), enable or disable the [panVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#panVisible) and [zoomVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#zoomVisible) properties. You can toggle switches under the map to see how these settings affect the control bar. 
```
